pkg/lib/cockpit-components-form-helper.jsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from "react";
+
+import { FormHelperText } from "@patternfly/react-core/dist/esm/components/Form/index.js";
+import { HelperText, HelperTextItem } from "@patternfly/react-core/dist/esm/components/HelperText";
+
+export const FormHelper = ({ helperText, helperTextInvalid, variant, icon, fieldId }) => {
+    const formHelperVariant = variant || (helperTextInvalid ? "error" : "default");
+
+    if (!(!!helperText || !!helperTextInvalid))
+        return null;
+
+    // FIXME: isHidden still adds some padding which is desired so that the form elements don't move when validation text appears
+    // Fix this seperately to avoid breaking pixe tests
+    return (
+        <FormHelperText isHidden={false}>
+            <HelperText>
+                <HelperTextItem
+                    id={fieldId ? (fieldId + "-helper") : undefined}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18641-20230415-073133-3760a938-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)